control-plane-operator/hostedclusterconfigoperator/controllers/resources/alerts/assets/podsecurity-violation.yaml

```diff
@@ -0,0 +1,40 @@
+# Source: https://raw.githubusercontent.com/openshift/cluster-kube-apiserver-operator/release-4.15/bindata/assets/alerts/podsecurity-violation.yaml
```

Comment:
This source link is wrong.  In needs to be https://raw.githubusercontent.com/openshift/cluster-kube-apiserver-operator/release-4.15/bindata/assets/alerts/podsecurity-violations.yaml